swap/src/monero/wallet_rpc.rs

```diff
@@ -23,6 +23,9 @@ const DOWNLOAD_URL: &str = "https://downloads.getmonero.org/cli/monero-mac-x64-v
 #[cfg(all(target_os = "macos", target_arch = "arm"))]
 const DOWNLOAD_URL: &str = "https://downloads.getmonero.org/cli/monero-mac-armv8-v0.18.0.0.tar.bz2";
 
+#[cfg(all(target_os = "macos", target_arch = "aarch64"))]
+const DOWNLOAD_URL: &str = "https://downloads.getmonero.org/cli/monero-mac-armv8-v0.18.0.0.tar.bz2";
+
```

Comment:
Oops, this is what I was trying to do above but got the target arch wrong 😅 

So I think lines 23 - 25 can be removed.

Reply:
ah you're right